ui/js/component/inviteNew/view.jsx

```diff
@@ -27,7 +27,7 @@ class FormInviteNew extends React.PureComponent {
     const { errorMessage, isPending } = this.props;
 
     return (
-      <form>
+      <form onSubmit={event => this.handleSubmit(event)}>
```

Comment:
If every form needs to call `event.preventDefault()` to prevent enter based submission from reloading the page, maybe we should just make a `Form` component that takes a required submit property? 

Then this component can handle preventing the event, without every individual form being required to remember to add this.

Reply:
Yeah, I should have done that... :stuck_out_tongue: 
 here?  -> https://github.com/lbryio/lbry-app/blob/master/ui/js/component/form.js

Reply:
maybe move that in to `components/forms` ?